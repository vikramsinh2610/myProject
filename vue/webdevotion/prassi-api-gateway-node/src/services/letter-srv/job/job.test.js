const Job = require('./job');

describe('Job', () => {
  test('should have these properties', () => {
    const job = new Job({
      roleId: 'PROMOTER',
    });
    expect(job).toHaveProperty('roleId');
  });

  test('should return its JSON Schema', () => {
    expect(Job.getJSONSchema()).toMatchSnapshot();
  });
});
