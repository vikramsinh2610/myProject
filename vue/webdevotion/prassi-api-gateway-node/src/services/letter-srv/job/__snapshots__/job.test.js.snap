// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Job should return its JSON Schema 1`] = `
Object {
  "properties": Object {
    "roleId": Object {
      "type": "string",
    },
  },
  "required": Array [
    "roleId",
  ],
  "type": "object",
}
`;
